templates/_helpers.yaml

```diff
@@ -160,6 +160,7 @@
 [pgbouncer]
 pool_mode = transaction
 listen_port = {{ .Values.ports.pgbouncer }}
+server_tls_sslmode = {{ .Values.pgbouncer.serverTlsSslmode }}
```

Comment:
@andriisoldatenko do we may need to add a `.Values.pgbouncer.serverTlsSslmode | default "prefer"` here to support a clean upgrade from a version of the chart that does not include this value.

You and @sjmiller609 did some work here so not sure if we still need to do this or not. What do you think?

Reply:
@schnie so do you think this won't work?
https://github.com/astronomer/airflow-chart/pull/101/files#diff-ba05023d0f1b5d4b64b6cad4cf9a7ecdR313